test/api.js

```diff
@@ -0,0 +1,29 @@
+import events from './data/events';
```

Comment:
I like this approach of having a centralized set of test events that we can then access through mocked API methods, should avoid a lot of repetitive data mocking 👍